### ISSUE TITLE
Show "Message data" in main winndow

### ISSUE THREAD

**Issue by humdingerb** [andimachovec/Kottan#2]
Instead of opening a window with the "Message data" when double-clicking an entry, Kottan could show it constantly on the right side of the main list view. Update when the selection changes. That will safe plenty of clicks: a double-click and click-to-close-the-window.

If you expect users will need to see the "Message data" of several entries side-by-side, a double-click could still open them in separate windows.

**Comment by andimachovec** [comment]
I was thinking about doing that but I worried that the list view would get too cluttered with data when there are many data sets under one label. But it is probably the better way to go, I'm not really that happy with the extra data window either.

**Comment by humdingerb** [comment]
Yeah, especially since it opens a third child window when editing... :)
